lib/transformers/tokend.js

```diff
@@ -14,7 +15,7 @@ function collectTransformables(properties, keyPath) {
   let results = [];
 
   // Ensure we're walking an Object. `== null` covers both null and undefined.
```

Comment:
I don't think this comment applies any more, since we're using `isPlainObject` instead of `source == null`.